src/data/config.rs

```diff
@@ -15,15 +15,24 @@ pub const CONFIG: &str = "config.yaml";
 
 const ADDR: &str = "0.0.0.0";
 const PORT: &str = "3000";
+const DEFAULT_FORGE_API_PAGE_SIZE: u8 = 10;
 
-#[derive(Serialize, Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct Config {
     /// The address of the server
     #[serde(skip_serializing_if = "Option::is_none")]
     addr: Option<String>,
     /// The port of the server
     #[serde(skip_serializing_if = "Option::is_none")]
     port: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    forges: Option<ConfigForges>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+struct ConfigForges {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    api_page_size: Option<u8>,
```

Comment:
I wonder if we should go for camelCase here instead of snake_case, since it's what e.g. kubernetes does. I dunno if there is any standards to follow here thou.

Reply:
We might use https://serde.rs/field-attrs.html#rename to achive this, so we can keep naming conventions consistent both in our codebase and outside in the config.

Reply:
...as in consistent with what yaml conventions and rust

Reply:
Cargo itself seems to be using snake_case: https://github.com/rust-lang/cargo/blob/master/src/cargo/sources/config.rs#L41

Clippy also says:

```
warning: structure field `fooBar` should have a snake case name
  --> src/data/config.rs:36:5
   |
36 |     fooBar: bool,
   |     ^^^^^^ help: convert the identifier to snake case: `foo_bar`
   |
   = note: `#[warn(non_snake_case)]` on by default
```

We could rename for serialization, indeed, but... I don't see why? There is no standard for configuration option naming, so I'd stick to having the same option name as in-code, so I don't have to remember that "oh, it's apiPageSize in yaml!".

Reply:
I think I've just been exposed to too many projects using camelCase, and come to expect it, but inspecting that expectation I do realize that is kinda silly since it doesn't seem to be standard at all. I say we go for snake case then.